app/scripts/modules/core/task/modal/comments.directive.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.task.comments.directive', [
+])
+  .directive('taskComments', function() {
+    return {
+      restrict: 'E',
+      scope: {
+        command: '=',
+      },
+      templateUrl: require('./comments.directive.html')
```

Comment:
When creating new directives, we've been avoiding attaching things directly to the `$scope` and instead using the `bindToController` option - even when there is no controller. It's a small set of changes to convert this code:
```javascript
return {
  restrict: 'E',
  scope: {},
  bindToController: {
    command: '=',
  },
  controller: angular.noop,
  controllerAs: 'vm',
  templateUrl: require('./comments.directive.html')
};
```
Then, in the directive itself, `ng-model="command.comments"` just becomes `ng-model="vm.command.comments"`.

We're doing this in the hopes that it makes migrating to Angular 2 easier when we start to tackle that.